test/expected/cursor.out

```diff
@@ -52,4 +52,4 @@ FETCH NEXT FROM c1;
 (1 row)
 
 UPDATE cursor_test SET temp = 0.7 WHERE CURRENT OF c1;
-COMMIT;
+ERROR:  cursor "c1" is not a simply updatable scan of table "cursor_test"
```

Comment:
are we sending a patch of this upstream?

Reply:
i'll put it on my todo list